index.js

```diff
@@ -1,110 +1,107 @@
-'use strict'
+'use strict';
 
 const koa = require('koa');
 const koaBody = require('koa-body');
 const GithubWebhookHandler = require('koa-github-webhook-handler');
 const github = require('./github');
 const bumpVersion = require('semver-increment');
+const promisify = require('es6-promisify');
 
-let app = koa();
+const app = koa();
 
-let githubWebhookHandler = new GithubWebhookHandler({
+const githubWebhookHandler = new GithubWebhookHandler({
   path: '/webhook',
   secret: process.env.SEANBOT_WEBHOOK_SECRET,
 });
 
+function getMasksFromCommentBody(commentBody) {
+  let MAJOR = 0;
+  let MINOR = 0;
+  let PATCH = 0;
+  if (commentBody.match('major')) {
+    MAJOR = 1;
+  } else if (commentBody.match('minor')) {
+    MINOR = 1;
+  } else if (commentBody.match('patch')) {
+    PATCH = 1;
+  } else {
+    return null;
+  }
+  const masks = [MAJOR, MINOR, PATCH];
+  return masks;
+}
+
+function getMsgFormPrUrl(url) {
+  const list = url.split('/');
+  return {
+    user: list[4],
+    repo: list[5],
+    number: list[7],
+  };
+}
+
+function getMsgFormHtmlUrl(url) {
+  const list = url.split('/');
+  return {
+    user: list[3],
+    repo: list[4],
+    branch: list[6],
+  };
+}
+
+const getPullRequests = promisify(github.pullRequests.get);
+const getContent = promisify(github.repos.getContent);
+const updateFile = promisify(github.repos.updateFile);
```

Comment:
To be more safe, use `promisify(github.repos.updateFile.bind(github.repos))`. The same to the above 2 lines :-)